intelmq/bots/outputs/smtp_batch/output.py

```diff
@@ -0,0 +1,354 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+#
+# Initiate dialog ex like this:
+# ssh -t $USER@proki.csirt.cz 'docker exec -it -u 1000 intelmq intelmq.bots.outputs.smtp_batch.output smtp-batch-output-cz cli
+#   --tester [your-email] --ignore-older-than-days 4'
+#
+from __future__ import unicode_literals
```

Comment:
This is already mandatory in Python >= 3.0

Reply:
Ok, thanks!